feature-flags/feature-flags.interface.ts

```diff
@@ -10,6 +10,7 @@ export interface FeatureFlagsInterface {
   FEATURE_TRANSPARENCY_COMMENTS: boolean; // disable Comments Tab
   FEATURE_RECOGNIZED_DELEGATES: boolean; // disable  Recognized Delegates page
   FEATURE_ACCOUNTS_SNAPSHOT: boolean; // disable Accounts Snapshot tab in the expense reports tab
-  FEATURE_ECOSYSTEM_ACTORS: boolean; // disable Accounts Snapshot page
+  FEATURE_ECOSYSTEM_ACTORS: boolean; // disable Actors Snapshot page
```

Comment:
Could you update the comment as this is not a "snapshot page"

Reply:
feature/ecosystem-actor-profile-page